tests/test_row_eq_len.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+
+def test_row_eq_len(prepare_tests):
+    tab, split, len_tab, cls = prepare_tests
+    failed_nums = []
+    for nmb in range(len_tab):
+        try:
+            table = cls.prepare_table(cls.data[split][nmb])
+            cells = table.get_cells()
+            same_length = all(len(row) == len(cells[0]) for row in cells)
+        except:
+            if not same_length:
```

Comment:
this should be outside of `except`, because the code can run fine without errors but if `same_length` is false, we should mark the table as failed. How did it work with this code? :D